pootle/templates/editor/_scripts.html

```diff
@@ -27,6 +27,7 @@
     isAdmin: {{ is_admin|yesno:"true,false" }},
     isAnonymous: {{ request.user.is_anonymous|yesno:"true,false" }},
     userId: {{ request.user.id }},
+    languageDirection: '{{ language.direction }}',
```

Comment:
To align with what has [already been done in #4171](https://github.com/julen/pootle/blob/e59ae110c95bef8f5b5cc0fc04ffde31ff08a360/pootle/static/js/editor/containers/Editor.js#L19), I'd suggest to name this `localeDir`.